yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1080,40 +1080,51 @@ func fetchUnsupportedObjectTypes() ([]UnsupportedFeature, error) {
 			unsupportedIndexes = append(unsupportedIndexes, ObjectInfo{
 				ObjectName: fmt.Sprintf("Index Name: %s, Index Type=%s", objectName, objectType),
 			})
-
+			// For oracle migration complexity comes from ora2pg, so defining Impact not required right now
 			assessmentReport.AppendIssues(AssessmentIssue{
-				Category:   UNSUPPORTED_FEATURES_CATEGORY,
-				Type:       "", // TODO
-				Name:       UNSUPPORTED_INDEXES_FEATURE,
-				ObjectType: "INDEX",
+				Category:            UNSUPPORTED_FEATURES_CATEGORY,
+				CategoryDescription: GetCategoryDescription(UNSUPPORTED_FEATURES_CATEGORY),
+				Type:                UNSUPPORTED_INDEXES_FEATURE,
+				Name:                UNSUPPORTED_INDEXES_ISSUE_TYPE,
```

Comment:
nit: I believe constant suffix with `_ISSUE_TYPE` is for `Type` field and `_FEATURE` is for `Name` field.

Reply:
yeah right, thanks.